openfisca_france/parameters/prestations/minima_sociaux/aspa/montant_annuel_seul.yaml

```
@@ -36,3 +36,5 @@ values:
   2020-01-01:
     value: 10838.40
     reference: https://www.legifrance.gouv.fr/eli/decret/2018/3/30/SSAS1805864D/jo/article_1
+  2021-01-01:
+    value: 10881.75
```

De même, circulaire CNAV : 
```suggestion
    value: 10881.75
    reference: https://www.legislation.cnav.fr/Documents/circulaire_cnav_2021_01_11012021.pdf
```